wallet/api/client_sign_transaction_test.go

```diff
@@ -66,7 +67,7 @@ func testSigningTransactionWithInvalidParamsFails(t *testing.T) {
 					"type": "not vega command",
 				},
 			},
-			expectedError: api.ErrTransactionIsNotValidVegaCommand,
+			expectedError: errors.New("the transaction is not a valid Vega command: unknown field \"type\" in vega.wallet.v1.SubmitTransactionRequest"),
```

Comment:
Does this mean that someone has used a field called 'type' and it's unknown?

Reply:
Just FYI, this a test. So I created dumb payload with a field `type`, knowing there is none for that object. When parsed as a transaction, proto is unhappy and tells me `unknown field \"type\"`. This is not a error message we have control of.

Reply:
Oh! Sorry I missed that it was a test, that explains why it's so specific